src/ipc/Kid.cc

```diff
@@ -21,6 +21,8 @@
 int TheProcessKind = pkOther;
 
 Kid::Kid():
+    processRole(),
```

Comment:
no need to explicitly call the SBuf constructor.

In related matters it maybe useful to make this class C++11 inititialized now and drop this constructor definition entirely, plus half of the next one.

Reply:
Made it C++11 initialized.